src/qml/editorwidgets/ExternalResource.qml

```diff
@@ -374,7 +374,7 @@ EditorWidgetBase {
     width: visible ? 48 : 0
     height: 48
 
-    visible: documentViewer == 0 && isEnabled
+    visible: (documentViewer == 0 || documentViewer == 3 ) && isEnabled
```

Comment:
Those values 0, 1, 2, ... for documentViewer could be declared as consts or enum for better redeability

Reply:
@domi4484 , they are documented in a comment towards the beginning of the QML: https://github.com/opengisch/QField/blob/70a063dc9fad99043edaec18b8bd8262de4ab1db/src/qml/editorwidgets/ExternalResource.qml#L63

Reply:
@nirvn ;)